parsely_raw_data/schema.py

```diff
@@ -65,6 +66,8 @@
     {"key": "metadata_thumb_url", "ex": "https://images.parsely.com/xY9xNBMulGDKRMzfKaUQzs7A9PA=/160x160/smart/http%3A//a.amz.mshcdn.com/media/ZgkyMDE2LzA5LzA3LzU2L0NyeFhpNjNYRUFBSnZwRS5lNDAyMy5qcGcKcAl0aHVtYgkxMjAweDYzMAplCWpwZw/156d0173/3ae/CrxXi63XEAAJvpE.jpg", "type": str, "available_with_field": "metadata"},
     {"key": "metadata_title", "ex": "Everyone has the same fear about Apple's new earbuds", "type": str, "available_with_field": "metadata"},
     {"key": "metadata_urls", "ex": ["http://mashable.com/2016/09/07/airpods-jokes/"], "type": list, "available_with_field": "metadata"},
+    {"key": "pageload_id", "ex": "66554433", "type": int},
```

Comment:
Can these 3 things be the same? (videostart, pageload, pageview ids)

Reply:
they could - but it would be unlikely - good catch I'll change them